### PR TITLE
update data in fragmentList  after reload page

### PR DESCRIPTION
**Pull request by SzajekSzymon**
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: src/js/components/FragmentList/FragmentList.jsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ import {
 import FragmentListItem from './FragmentListItem/FragmentListItem';
 import { FRAGMENT_LIST_HEADER, fragmentListTablesHeaders } from '../../helpers/constants';
 
-export function mapDataToComponents(fragments, tabId) {
+export const mapDataToComponents = (fragments, tabId) => {
```
Comment (MichalGraziowskiCognifide) [inline src/js/components/FragmentList/FragmentList.jsx:33]: Isn't eslint mad here? 
You could have performed a return directly after the arrow:
```
export const mapDataToComponents = (fragments, tabId) => fragments.map(({ debug, nodes }) => {
...
}
```

File: src/js/components/FragmentList/FragmentList.jsx [hunk 2]
```diff
@@ -72,14 +72,22 @@ const sortingOptions = {
...
   const [fragments, setFragments] = useState(parsedData);
   const [currentSorting, setCurrentSorting] = useState(null);
 
   const typeSortComparator = (a, b) => a.props.type.localeCompare(b.props.type);
   const idSortComparator = (a, b) => a.props.id.localeCompare(b.props.id);
   const timeSortComparator = (a, b) => a.props.time - b.props.time;
 
+  useEffect(() => {
+    parsedData = mapDataToComponents(data, tabId);
+    console.log(parsedData);
```
Comment (MichalGraziowskiCognifide) [inline src/js/components/FragmentList/FragmentList.jsx:86]: Rouge console.log 😉

File: src/js/components/FragmentList/FragmentList.jsx [hunk 3]
```diff
@@ -72,14 +72,22 @@ const sortingOptions = {
...
+    setFragments(parsedData);
```
Comment (xolir) [inline src/js/components/FragmentList/FragmentList.jsx:87]: I think this should be seperate function that sets fragment to parsedData and is resetting sort as this is used like 5 times inside this code

File: src/js/components/FragmentList/FragmentList.jsx [hunk 4]
```diff
@@ -72,14 +70,25 @@ const sortingOptions = {
...
+  const resetState = () => {
+    setFragments(parsedData);
+    setCurrentSorting(null);
+  };
+
   const typeSortComparator = (a, b) => a.props.type.localeCompare(b.props.type);
   const idSortComparator = (a, b) => a.props.id.localeCompare(b.props.id);
   const timeSortComparator = (a, b) => a.props.time - b.props.time;
 
+  useEffect(() => {
```
Comment (tomaszmichalak) [inline src/js/components/FragmentList/FragmentList.jsx:87]: Please try to implement unit test for this.

**Pull request merged**